### PR TITLE
Python testing fixes

### PR DESCRIPTION
**Pull request by EdNoepel**
To fix the real-world test and other things which utilize the brownie deployment:
- Update brownie testing to deploy `Auctions` library.
- Update parameter order quering min debt amount.
- Handle corner case where lender deposits when LUP is 0.

### CODE REVIEW

File: tests/brownie/sdk/ajna_protocol.py [hunk 1]
```diff
@@ -33,6 +34,7 @@ def __init__(self) -> None:
         self.deposits = Deposits.deploy({"from": self.deployer})
         self.maths = Maths.deploy({"from": self.deployer})
         self.loans = Loans.deploy({"from": self.deployer})
+        self.auctions = Auctions.deploy({"from": self.deployer})
```
Comment (EdNoepel) [inline tests/brownie/sdk/ajna_protocol.py:37]: Could not keep it alphabetical due to dependencies.

**Review by MikeHathaway — APPROVED** [review]
LGTM

File: tests/brownie/test_stable_volatile.py [hunk 2]
```diff
@@ -195,7 +195,7 @@ def pledge_and_borrow(pool_helper, borrower, borrower_index, collateral_to_depos
     if not ensure_pool_is_funded(pool, borrow_amount, "borrow"):
         # ensure_pool_is_funded logs a message
         return
-    (_, _, _, min_debt) = pool_helper.utilizationInfo()
+    (min_debt, _, _, _) = pool_helper.utilizationInfo()
```
Comment (EdNoepel) [inline tests/brownie/test_stable_volatile.py:198]: param order changed

File: tests/brownie/test_stable_volatile.py [hunk 3]
```diff
@@ -303,7 +303,8 @@ def draw_debt(borrower, borrower_index, pool_helper, test_utils, collateralizati
 def add_quote_token(lender, lender_index, pool_helper):
     dai = pool_helper.quoteToken()
     index_offset = ((lender_index % 6) - 2) * 2
-    deposit_index = pool_helper.lupIndex() - index_offset
+    lup_index = pool_helper.lupIndex()
+    deposit_index = lup_index - index_offset if lup_index > 6 else MAX_BUCKET
```
Comment (EdNoepel) [inline tests/brownie/test_stable_volatile.py:307]: Fix bug handling deposit after all deposit has been removed and LUP is 0.
